src/main/resources/view/MainWindow.fxml

```diff
@@ -2,15 +2,14 @@
 
 <?import java.net.URL?>
 <?import javafx.geometry.Insets?>
-<?import javafx.scene.Scene?>
 <?import javafx.scene.control.Menu?>
 <?import javafx.scene.control.MenuBar?>
 <?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.control.SplitPane?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
-
+<?import javafx.scene.Scene?>
+<?import javafx.stage.Stage?>
```

Comment:
Not sure why stage was imported

Reply:
I think it is because the next line has type="javafx.stage.Stage" and auto-import include it as a result. rather insignificant though.